Products/Poi/events.py

```diff
@@ -86,7 +85,11 @@ def remember_tracker(new_issue, event):
 
 
 def assign_id(new_issue, event):
-    """Auto-increment ID numbers"""
+    """Auto-increment ID numbers when they are created.
+       Don't run on copied issues
+    """
+    if new_issue.id.find('copy') >= 0:
+        return
```

Comment:
I know it’s an edge case, but if you were moving an issue to a different tracker, would this prevent it from getting a new valid ID?